services/core-web/src/components/Forms/noticeOfWork/EditNoticeOfWorkDocumentForm.js

```diff
@@ -65,10 +65,22 @@ export class EditNoticeOfWorkDocumentForm extends Component {
 
   render() {
     const filteredDropDownOptions = this.props.dropdownNoticeOfWorkApplicationDocumentTypeOptions.filter(
-      ({ value }) =>
-        this.props.categoriesToShow.length > 0 ? this.props.categoriesToShow.includes(value) : value
+      ({ subType, value }) => {
+        if (subType && this.props.categoriesToShow.length > 0) {
+          return this.props.categoriesToShow.includes(subType);
+        }
+        if (this.props.categoriesToShow.length > 0) {
+          return this.props.categoriesToShow.includes(value);
+        }
+        return true;
+      }
```

Comment:
This needs to be an array not a boolean - so if there is nothing to filter, return the entire set